test-avocado/compiletest.sh

```diff
@@ -16,7 +19,7 @@ test -d $PACKAGE/$BUILD && /bin/rm -r $PACKAGE/$BUILD
 cd $PACKAGE
 mkdir -p $BUILD
 cd $BUILD
-../autogen.sh --prefix=/usr --enable-maintainer-mode --enable-debug
+../autogen.sh --prefix=/usr --libdir=/usr/lib$ARCH64 --enable-maintainer-mode --enable-debug
```

Comment:
Hmmm, why is this not using tools/make-rpms? That contains a lot of logic about what should be installed.